android/src/main/kotlin/com/apptreesoftware/mapview/MapViewPlugin.kt

```diff
@@ -95,6 +96,7 @@ class MapViewPlugin(val activity: Activity) : MethodCallHandler {
                 toolbarActions = getToolbarActions(call.argument<List<Map<String, Any>>>("actions"))
                 showUserLocation = mapOptions["showUserLocation"] as Boolean
                 mapTitle = mapOptions["title"] as String
+                mapViewType = mapOptions["mapViewType"] as Int
```

Comment:
Not guarantee to not be null currently. You should cast it to Int? and check to make sure it's not null before setting it on `mapViewType`.